.gitpod.yml

```diff
@@ -0,0 +1,27 @@
+# Configure a custom docker image. Here we install 'cargo watch'
+image:
+  file: Dockerfile
```

Comment:
I'd like to keep the main directory as clean as possible so maybe we can move the Dockerfile to be something like `examples/util/gitpod.Dockerfile`

Reply:
I went with `.gitpod/Dockerfile` which is another convention